accounts/safe7579/src/zkEmailRecovery/modules/SafeRecoveryModule.sol

```diff
@@ -103,25 +104,13 @@ contract SafeRecoveryModule is RecoveryModuleBase {
             revert InvalidNewOwner();
         }
 
-        bool accountIsConfigured = accountIsConfigured[account];
-        if (!accountIsConfigured) {
-            revert AccountNotConfigured();
-        }
-
-        address previousOwnerInLinkedList = getPreviousOwnerInLinkedList(
-            account,
-            oldOwner
-        );
-        bytes memory encodedSwapOwnerCall = abi.encodeWithSignature(
-            "swapOwner(address,address,address)",
-            previousOwnerInLinkedList,
-            oldOwner,
-            newOwner
-        );
-        IERC7579Account(account).executeFromExecutor(
```

Comment:
ExecutorBase exposes an internal function to structure simple calls. I think this improves readability a bit